src/modules/core/components/emoji/emoji.tsx

```diff
@@ -0,0 +1,28 @@
+import { type FC } from 'react';
+// animations
+import Lottie from 'lottie-react';
+import sunglasses from '@/assets/lottie-files/emoji/emoji-sunglasses.json';
+import heartEyes from '@/assets/lottie-files/emoji/emoji-heart-eyes.json';
+
+// keys
+const emojies = {
+  sunglasses,
+  heart_eyes: heartEyes,
+} satisfies Record<string, unknown>;
```

Comment:
The type unknown is bothering me, maybe we can write some type that will deskribe it better?

Reply:
We have icon name - `image-not-found`, so suggest use same principe - `heart-eyes`.

Reply:
we can put `object` instead of `unknown` because there is no static structure for `animationData`